Makefile

```diff
@@ -132,9 +132,18 @@ cxx-libraries:
 cxx-test-%: force
 	@$(MAKE) -f mk/cxx.mk --no-print-directory CXX_MODE=$(CXX_MODE) CXX_DIR=$(CXX_DIR) $@
 
+GLEAN_CABAL_BINS=glean glean-server glean-hyperlink
+GLEAN_NATIVE_LIBS=librocksdb.so.8 libfolly.so.0.58.0-dev libfmt.so.11
```

Comment:
this looks fragile, can we make it independent of the version numbers?

Reply:
Maybe we can just do something like `librocksdb.so.*` or something like that and let `bash` do the rest of the magic for us

Reply:
I looked into this but it's not quite that simple, because just copying everything with an asterisk copies way too much and bloats up the container again. Instead, I think I'll just make sure that the built docker container has test queries run against it, which should ensure that the proper shared libraries are available (otherwise it wouldn't launch), so at least a break will be obvious and reported.